salt/ufw/etc_ufw/sysctl.conf

```diff
@@ -32,10 +56,42 @@ net/ipv4/conf/default/log_martians=0
 #net/ipv4/tcp_fin_timeout=30
 #net/ipv4/tcp_keepalive_intvl=1800
 
+# Ignore Directed pings
+#net/ipv4/icmp_echo_ignore_all = 1
+
 # Uncomment this to turn off ipv6 autoconfiguration
 #net/ipv6/conf/default/autoconf=1
 #net/ipv6/conf/all/autoconf=1
 
 # Uncomment this to enable ipv6 privacy addressing
 #net/ipv6/conf/default/use_tempaddr=2
 #net/ipv6/conf/all/use_tempaddr=2
+
+# Performance Recommended Settings:
```

Comment:
This is 1) specific to AWS 2) "take only when prescribed from performance engineer" 3) "best before 2015" and most importantly 4) unlikely to get any benefit in this scenario.

I am not going to merge the performance part, please remove it from this PR.